File: samples_generated_code/samples_1/phi4/baseline_code_java/code_row_64.java
import java.util.Scanner;

public class ArrayIndexValue {

    private static final int[] ID_SEQUENCE = {10, 20, 30, 40, 50, 60, 70, 80, 90, 100};

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter an index value: ");
        int index = scanner.nextInt();
        
        try {
            int value = getValueAtIndex(index);
            System.out.println("Value at index " + index + " is: " + value);
        } catch (ArrayIndexOutOfBoundsException e) {
            System.out.println("Invalid index. Please enter a value between 0 and " + (ID_SEQUENCE.length - 1) + ".");
        }
        
        scanner.close();
    }

    public static int getValueAtIndex(int index) {
        return ID_SEQUENCE[index];
    }
}

File: samples_generated_code/sample_3/phi4/baseline_code_py/code_row_64.py
def get_value_at_index(index):
    id_sequence = [10, 20, 30, 40, 50, 60, 70, 80, 90, 100]
    return id_sequence[index]

def main():
    index = int(input("Enter an index value: "))
    value = get_value_at_index(index)
    print(f"The value at index {index} is {value}")

if __name__ == "__main__":
    main()